src/Stratis.Bitcoin.IntegrationTests/Wallet/WalletTests.cs

```diff
@@ -67,7 +62,7 @@ public void WalletCanReceiveAndSendCorrectly()
                 Assert.Null(stratisReceiver.FullNode.WalletManager().GetSpendableTransactionsInWallet(WalletName).First().Transaction.BlockHeight);
 
                 // Generate two new blocks so the transaction is confirmed
-                TestHelper.GenerateBlockManually(stratisSender, new List<Transaction>(new[] { stratisSender.FullNode.Network.CreateTransaction(trx.ToBytes()) }));
+                stratisSender.AddToStratisMempool(trx);
```

Comment:
Is that the same behavior?

Reply:
Kind of yes, there is no difference between generating a block manually and including a transaction as opposed to just adding the tx to the mempool and mine it normally.

Reply:
ok sounds good